R/compose.R

```diff
@@ -2,6 +2,9 @@
 #'
 #' @param ... Functions to apply in order from right to left. Formulas
 #'   are converted to functions in the usual way.
+#'
+#'   These dots support [tidy dots features][rlang::list2]. In particular, you can
+#'   splice lists of functions with `!!!`.
```

Comment:
I would re-word to clarify who's doing what.

Proposal: "These dots support tidy dots features. In particular, if you pass a list of functions, it will be spliced with `!!!`.

Reply:
Done, thanks!